glotaran/utils/io.py

```diff
@@ -186,3 +186,41 @@ def relative_posix_path(source_path: StrOrPath, base_path: StrOrPath | None = No
         except ValueError:
             pass
     return Path(source_path).as_posix()
+
+
+def safe_parameters_fillna(df, column_name, fill_value):
```

Comment:
This function needs tests.

Reply:
Added by @patrickhaetti in 7233dbce5540b769a520b8e648b3be669e5deafb